docs/user_guide/preemption.md

```diff
@@ -0,0 +1,252 @@
+6. A task can only preempt a task with lower or equal priority
+7. A task cannot preempt tasks outside its preemption fence
+
+For a detailed explanation of these preemption laws, please refer to the preemption [design document](design/preemption.md#the-laws-of-preemption).
+
+Next, we will provide a few examples to help you understand the functionality and impact of preemption, allowing you to deploy it effectively in your environment. You can find the necessary files for the examples in the yunikorn-k8shim/deployment/example/preemption directory.
+
+Included in the files is a YuniKorn configuration that defines the queue configuration as follows:
+
+```bash
+queues.yaml: |
+    partitions: 
+    - name: default
+      placementrules:
+        - name: provided
+          create: true
+      queues:
+        - name: root
+          submitacl: '*'
+          properties:
+            preemption.policy: fence
+            preemption.delay: 10s
+          queues:
+          - name: 1-normal ...
+          - name: 2-no-guaranteed ...
+          - name: 3-priority-class ...
+          - name: 4-priority-queue ...
+          - name: 5-fence ...
+```
+
+Each queue corresponds to a different example, and the preemption will be triggered 10 seconds after deployment, as indicated in the configuration `preemption.delay: 10s`.
+
+### General Preemption Case
+
+In this case, we will demonstrate the outcome of triggering preemption when the queue resources are distributed unevenly in a general scenario.
+
+We will deploy 10 pods with a resource requirement of 1 to both `queue-1` and `queue-2`. First, we deploy to `queue-1` and then introduce a few seconds delay before deploying to `queue-2`. This ensures that the resource usage in `queue-1` will exceed that of `queue-2`, depleting all resources in the parent queue and triggering preemption.
+
+| Queue            | Max Resource | Guaranteed Resource |
+| ---------------- | ------------ | ------------------- |
+| `normal`         | 12           | - (not configured)  |
+| `normal.queue-1` | 10           | 5                   |
+| `normal.queue-2` | 10           | 5                   |
+
+Result:
+
+When a set of guaranteed resources is defined, preemption aims to ensure that all queues satisfy their guaranteed resources. Preemption stops once the guaranteed resources are met (law 4). A queue may be preempted if it has more resources than its guaranteed amount. For instance, in this case, if queue-1 has fewer resources than its guaranteed amount (<5), it will not be preempted (law 5).
+
+| Queue            | Resource before preemption | Resource after preemption              |
+| ---------------- | -------------------------- | -------------------------------------- |
+| `normal.queue-1` | 10 (victim)                | 7                                      |
+| `normal.queue-2` | 2                          | 5 (guaranteed resources are satisfied) |
```

Comment:
Formatting issues: Change `(guaranteed resources are satisfied)` to `(guaranteed minimum)`. This will avoid wrapping the queue name.